### PR TITLE
Upgrade protobuf and grpcio 

### PR DESCRIPTION
**Pull request by kse3hi**
## Describe your changes

Update grpcio to avoid security issues

## Issue ticket number and link
ADO#15657

## Checklist - Manual tasks

* [x] Examples are executing successfully
* [ ] Created/updated unit tests. Code Coverage percentage on new code shall be >= 80%.
* [ ] Created/updated integration tests.
* [x] Devcontainer can be opened successfully
* [x] Devcontainer can be opened successfully behind a corporate proxy
* [x] Devcontainer can be re-built successfully
* [ ] Extended the documentation (e.g. README.md, CONTRIBUTING.md, Velocitas)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -70,7 +68,7 @@
 
 setup(
     name="sdv",
-    version="0.10.1",
+    version="0.11.0",
```
Comment (BjoernAtBosch) [inline setup.py:71]: Shouldn't be a "0.10.2" enough? I don't see new features or even incompatibilities ...

File: .vscode/launch.json [hunk 2]
```diff
@@ -17,7 +17,8 @@
                 "SDV_MQTT_ADDRESS": "mqtt://localhost:1883",
                 "SDV_HVACSERVICE_ADDRESS": "grpc://localhost:50052",
                 "SDV_SEATSERVICE_ADDRESS": "grpc://localhost:50051",
-                "SDV_VEHICLEDATABROKER_ADDRESS": "grpc://localhost:55555"
+                "SDV_VEHICLEDATABROKER_ADDRESS": "grpc://localhost:55555",
+                "PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION": "python"
```
Comment (BjoernAtBosch) [inline .vscode/launch.json:21]: Why is that needed?

**Review by BjoernAtBosch — COMMENTED** [review]
Some questions to be clarified

File: .vscode/launch.json (hunk 2 above)
Comment (BjoernAtBosch) [reply .vscode/launch.json:21]: Ok - had a fast googling about it: As far as I understood another solution would be to increase protobuf to version 3.20.x (see https://github.com/protocolbuffers/protobuf/issues/10051).
(In C++ SDK we use protobuf 3.21.9)

Comment (kse3hi) [reply .vscode/launch.json:21]: grpcio is required protobuf >4

File: setup.py (hunk 1 above)
Comment (kse3hi) [reply setup.py:71]: Especially because of protobuf 3 --> 4

File: .vscode/launch.json (hunk 2 above)
Comment (BjoernAtBosch) [reply .vscode/launch.json:21]: My question related to the env var PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION and why we need that.
In my "self-reply" above I mentioned that people in the linked thread said, that this variable isn't required for protobuf > 3.20.x

File: setup.py (hunk 1 above)
Comment (BjoernAtBosch) [reply setup.py:71]: Ok, but does that mean a "big change" for our users? Or: Do they even recognize?

File: .vscode/launch.json (hunk 2 above)
Comment (kse3hi) [reply .vscode/launch.json:21]: @BjoernAtBosch , advice is to downgrade protobuf to 3.20 and low....
image

In this case it will be not possible to update grpcio which include security fixes for 6 vulnerabilities. grpcio >= 1.53 is required protobuf < 4

It means that only env variable is suitable in our case.

Comment (BjoernAtBosch) [reply .vscode/launch.json:21]: Ah - yes, you're right. I was puzzled by this somewhat confusing versioning scheme of protobuf.

File: setup.py (hunk 1 above)
Comment (BjoernAtBosch) [reply setup.py:71]: Yes, but isn't that just needed for the stub generation - or generally during "runtime"?

**Review by BjoernAtBosch — DISMISSED** [review]
LGTM
